src/Umbraco.Core/Persistence/Repositories/MemberRepository.cs

```diff
@@ -696,12 +696,12 @@ private IEnumerable<IMember> ProcessQuery(Sql sql)
                 d.dto.ContentVersionDto.ContentDto.NodeDto.CreateDate,

```

Comment:
this `d.dto.ContentVersionDto.ContentDto.NodeDto`  seems to always end up being null which throws an exception - as seen in the MemberRepositoryTests